src/test/java/com/mael/payroll/paymentSchedules/MonthlyScheduleTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mael.payroll.paymentSchedules;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static java.time.Month.JANUARY;
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class MonthlyScheduleTest {
+
+    private MonthlySchedule monthlySchedule;
+
+    @Before
+    public void setUp() {
+        monthlySchedule = new MonthlySchedule();
+    }
+
+    @Test
+    public void returnsTrueIfLastDayOfMonth() {
```

Comment:
The `returnsTrue...` and `returnsFalse...` reads a bit too technical for me.
Is there a way to express the behaviour you're verifying in a way that is it more descriptive?
In the end test names act as documentation for us and our team.